template/cordova/lib/PluginHandler.js

```diff
@@ -53,18 +53,37 @@ var handlers = {
     },
     'resource-file':{
         install:function(obj, plugin, project, options) {
-            // do not copy, but reference the file in the plugin folder. This allows to
-            // have multiple source files map to the same target and select the appropriate
-            // one based on the current build settings, e.g. architecture.
-            // also, we don't check for existence. This allows to insert build variables
-            // into the source file name, e.g.
-            // <resource-file src="$(Platform)/My.dll" target="My.dll" />
-            var relativeSrcPath = getPluginFilePath(plugin, obj.src, project.projectFolder);
-            project.addResourceFileToProject(relativeSrcPath, obj.target, getTargetConditions(obj));
+            var targetConditions = getTargetConditions(obj);
+            if (targetConditions.reference) {
```

Comment:
I think that the object that `getTargetConditions` returns was originally meant to hold details about target platform (Win8.1, WP8.1, Win10) and architecture to create a `condition` attribute, so `reference` field  doesn't really fit here - you might just check `if (obj.reference) ...`